packages/compiler/package.json

```diff
@@ -68,12 +68,11 @@
     "change-case": "~4.1.2"
   },
   "devDependencies": {
-    "@types/glob": "~7.1.3",
     "@types/js-yaml": "~4.0.1",
     "@types/mkdirp": "~1.0.1",
     "@types/mocha": "~9.1.0",
     "@types/mustache": "~4.1.2",
-    "@types/node": "~14.0.27",
+    "@types/node": "~16.11.7",
```

Comment:
I think it's best to keep 16.0.x so we don't take a dependency on an API added in minor release and we can keep our requirment just "node 16". Basically, you can revert this after merging with main.

Reply:
sounds good, just taking up your change instead of this.